airgun/views/contentview_new.py

```diff
@@ -0,0 +1,62 @@
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import ExpandableTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixinPF4
+
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixinPF4):
+    title = Text('.//h1[contains(@data-ouia-component-id, "OUIA-Generated-Text")]')
+    create_content_view = PF4Button('create-content-view')
+    table = ExpandableTable(
+        component_id='content-views-table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Last Published': Text('./a'),
+            'Last task': Text('.//a'),
+            'Latest version': Text('.//a'),
+        },
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.create_content_view, exception=False) is not None
```

Comment:
Its not a very good pattern to use `wait_for_element` within `is_displayed`, as `is_displayed` gets wrapped by `wait_displayed`.  So you end up stacking wait conditions.

Separately its good to check multiple widgets on the page, not just one. Does the page only show the create_content_view button when there are no content views? is the table always displayed or only after the first content view is created?

Reply:
The table is displayed only after the cv is created, otherwise there's no table.  And the page shows the `create_content_view` button regardless if there's a cv or not.